src/services/storage-options/restApiService.js

```diff
@@ -99,15 +99,32 @@ export default class RestApiService extends StorageInterface {
   async updateConfig(config) {
     const response = await axios.put(
       `http://localhost:5000/v1/app/org/f352ae63-11fc-4dbe-bab1-72561aa25fca/config/5e0f71ff-987d-4240-85eb-df6adf568c31`,
-      { config }
+      { config, withCredentials: true }
```

Comment:
You can't hardcode this here because they won't always be using cookie auth. They can still use a bearer token as well. Please use YattIntegrationHelpers.getHeaders.